pkg/plugin/config/config_loader_test.go

```diff
@@ -1,16 +1,18 @@
-package config
+package config_test
 
 import (
+	"github.com/arquillian/ike-prow-plugins/pkg/plugin/config"
 	"github.com/arquillian/ike-prow-plugins/pkg/scm"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	gock "gopkg.in/h2non/gock.v1"
+	"gopkg.in/h2non/gock.v1"
```

Comment:
Please don't remove that `gock "gopkg.in/h2non/gock.v1"` - as noted in the [docs](http://arquillian.org/ike-prow-plugins/#_building) if we use `goimports` this will fail. Is that IDEA/Goland doing it on your behalf?

Reply:
I ran `goimports` before I pushed, so I thought that it would complain about it.

Reply:
It might be that it's not stripping the tests. I'm a bit confused with this tool now :)

Reply:
so, it probably doesn't validate the tests...

Reply:
d062bb8